apps/buildog/src/app/organizations/[organizationName]/settings/page.tsx

```diff
@@ -155,15 +153,17 @@ export default function Page() {
             </form>
           </Form>
         </TabsContent>
-        <TabsContent value="org">
+        <TabsContent value="users">
           <div className="flex items-center justify-between">
             <div className="space-y-1">
-              <h2 className="text-2xl font-semibold tracking-tight">Your Organizations</h2>
-              <p className="text-sm text-muted-foreground">Manage your organizations.</p>
+              <h2 className="text-2xl font-semibold tracking-tight">Users</h2>
+              <p className="text-sm text-muted-foreground">Users signed in to this organization.</p>
             </div>
           </div>
           <Separator className="my-4" />
-          <Organizations />
+          <p>
+            Users signed in for this organization will appear here after connecting the backend.
+          </p>
```

Comment:
We don't need this text. Can you add an empty data component here for "User".

![Screenshot 2024-09-03 at 10 26 11 PM](https://github.com/user-attachments/assets/4fc3887b-9d0c-4c88-ac58-08abc9bbf85c)